### ISSUE TITLE
add changelog notes of 1.2.4

### ISSUE THREAD

**Issue by cwmp** [ElemeFE/element#3472]
### Steps to reproduce
http://element.eleme.io/#/en-US/component/changelog

### What is Expected?

expected changelog notes of latest version 1.2.4

### What is actually happening?

changelog of older version 1.2.3 is displayed instead

**Comment by Leopoldthecoder** [comment]
Please clear browser storage.

**Comment by cwmp** [comment]
Thanks. That actually worked, but I am wondering why it happend. I used the latest Chrome Version on two different machines with the same result.. just saw the changelog of 1.2.3. Even performing CTRL+F5 did not invalidate the page cache but served it from cache (disk vom cache). The Chrome Dev Tools revealed that clearly.

Here the some details of the request (Dev Tools) from the one machine where I did not cleared the cache yet.

General:
Request URL:http://element.eleme.io/
Request Method:GET
Status Code:200 OK (from disk cache)
Remote Address:151.101.64.133:80
Referrer Policy:no-referrer-when-downgrade

Response Headers:

Accept-Ranges:bytes
Access-Control-Allow-Origin:*
Age:38
Cache-Control:max-age=600
Content-Encoding:gzip
Content-Length:759
Content-Type:text/html; charset=utf-8
Date:Wed, 01 Mar 2017 19:43:27 GMT
Expires:Wed, 01 Mar 2017 19:34:43 GMT
Last-Modified:Wed, 01 Mar 2017 07:30:15 GMT
Server:GitHub.com
Vary:Accept-Encoding
Via:1.1 varnish
X-Cache:HIT
X-Cache-Hits:1
X-Fastly-Request-ID:12661b85083e50ff2fc539c4123f958fd3459606
X-GitHub-Request-Id:F66E:868A:3EC9CFB:532C676:58B71FFB
X-Served-By:cache-hhn1544-HHN
X-Timer:S1488397407.086398,VS0,VE0
